app/src/main/java/com/github/se/signify/ui/screens/home/QuizScreen.kt

```diff
@@ -57,72 +54,45 @@ fun QuizScreen(navigationActions: NavigationActions, quizRepository: QuizReposit
 
   val context = LocalContext.current
 
-  Scaffold(
-      modifier = Modifier.fillMaxSize(),
-  ) { padding ->
-    Column(
-        modifier = Modifier.fillMaxSize().padding(16.dp),
-    ) {
-      QuizHeader(navigationActions)
-
-      Spacer(modifier = Modifier.height(24.dp))
-
-      if (currentQuiz != null) {
-        val shuffledOptions =
-            remember(currentQuiz) {
-              currentQuiz!!.confusers.plus(currentQuiz!!.correctWord).shuffled()
-            }
-
-        QuizContent(
-            currentQuiz = currentQuiz!!,
-            shuffledOptions = shuffledOptions,
-            selectedOption = selectedOption,
-            onOptionSelected = { selectedOption = it },
-            onSubmit = {
-              quizViewModel.submitAnswer(
-                  selectedOption = it,
-                  onCorrect = {
-                    Toast.makeText(context, "Correct answer!", Toast.LENGTH_SHORT).show()
-                  },
-                  onIncorrect = {
-                    Toast.makeText(context, "Incorrect answer, try again.", Toast.LENGTH_SHORT)
-                        .show()
-                  })
-              selectedOption = null
-            })
-      } else {
-        NoQuizAvailable()
-      }
+  AnnexScreenScaffold(navigationActions = navigationActions, testTagColumn = "QuizScreen") {
+    Text(
+        text = "Quiz Time !",
+        fontWeight = FontWeight.Bold,
+        fontSize = 30.sp,
+        color = MaterialTheme.colorScheme.primary,
+        textAlign = TextAlign.Center,
+        modifier = Modifier.fillMaxWidth().testTag("QuizTitle"))
+
+    Spacer(modifier = Modifier.height(24.dp))
+
+    if (currentQuiz != null) {
+      val shuffledOptions =
+          remember(currentQuiz) {
+            currentQuiz!!.confusers.plus(currentQuiz!!.correctWord).shuffled()
```

Comment:
```suggestion
            currentQuiz.confusers.plus(currentQuiz.correctWord).shuffled()
```
Removing the !! operator improves code safety by eliminating the risk of runtime exceptions caused by null assertions. It also makes the code cleaner especially when null checks already guarantee safety by the if condition.

Reply:
By applying the change requested we got the following error:

![image](https://github.com/user-attachments/assets/99ee90ad-5c55-4a4d-8ac9-1b1bd33c3612)

I think it is best to let it as it was.